hubspot/management/commands/sync_hubspot.py

```diff
@@ -41,6 +41,13 @@ def bulk_sync_model(objects, make_object_sync_message, object_type, **kwargs):
         sync_messages = [
             make_object_sync_message(obj.id, **kwargs)[0] for obj in objects
         ]
+
+        if object_type == "CONTACT":
+            # Skip sync if message is missing required field
+            sync_messages = [
+                message for message in sync_messages if message.get('propertyNameToValues', {}).get('email')
```

Comment:
Just noting: this assumes that if the email field is populated, all the other fields we care about are there, too. A safe assumption if the only reason email isn't there is if the data population didn't happen.